manheim_cloudmapper/ses/test/test_report.py

```diff
@@ -0,0 +1,259 @@
+# Copyright 2017-2019 Manheim / Cox Automotive
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+import datetime
+from manheim_cloudmapper.ses.report import Report
+
+# https://code.google.com/p/mock/issues/detail?id=249
+# py>=3.4 should use unittest.mock not the mock package on pypi
+if (
+        sys.version_info[0] < 3 or
+        sys.version_info[0] == 3 and sys.version_info[1] < 4
+):
+    from mock import patch, call, Mock, mock_open
+else:
+    from unittest.mock import patch, call, Mock, mock_open
```

Comment:
Same comment here as on one of the other test files... we're not supporting Python < 3.5, so lines 19-26 can go away.